### PR TITLE
Prevent AutoHotkey script from intercepting `Ctrl+Tab` in VSCode/VSCodium

### PR DESCRIPTION
**Pull request by eternalphane**

### CODE REVIEW

File: docs/ctrl-tab/quickey-ctrl-tab.ahk [hunk 1]
```diff
@@ -22,6 +22,23 @@ MinUpDownTicks            := 200
...
 SawCtrlTab                := 0
 
+ExcludeTitles             := ["(Visual Studio Code|VSCodium)( - Insiders)?$"] ; VSCode / VSCodium
+
+ExcludeTitle := ExcludeTitles[1]
+For Idx, Title in ExcludeTitles {
+	If (Idx > 1) {
+		ExcludeTitle .= "|" . Title
+	}
+}
```
Comment (fwextensions) [inline docs/ctrl-tab/quickey-ctrl-tab.ahk:32]: I'm not really an expert in AutoHotkey.  What is this looping over?  It kind of looks like `ExcludeTitles` is a regex that might match multiple app names, but then it ends up with just a single `ExcludeTitle` string.  I guess it's building up a string of OR'd titles?

**Comment by fwextensions** [comment]
Thanks for pointing this out, and including a fix!  I had a question about how the script works.

File: docs/ctrl-tab/quickey-ctrl-tab.ahk (hunk 1 above)
Comment (eternalphane) [reply docs/ctrl-tab/quickey-ctrl-tab.ahk:32]: > I guess it's building up a string of OR'd titles?

You are right. I think extending an array of OR'd regexes can be much easier than maintaining an all-in-one regex.

Comment (fwextensions) [reply docs/ctrl-tab/quickey-ctrl-tab.ahk:32]: NVM.  I think I understand it now.  Seems to work well!

**Pull request merged**